gravity-forms/gw-limit-multiselect.js

```diff
@@ -4,26 +4,35 @@
  *
  * Limit how many options may be selected in a Multi Select field. Works with
  * regular Multi Select fields as well as fields with Enhanced UI enabled.
- * 
+ *
  * Instructions:
- * 
+ *
  * 1. Install this snippet with our free Custom JavaScript plugin.
  *    https://gravitywiz.com/gravity-forms-custom-javascript/
  * 2. Configure snippet per inline instructions.
  */
-// Update "1" to the ID of your Multi Select field.
-$( '#input_GFFORMID_1' ).on( 'change', function () {
-	// Update "2" to the max number of options that should be selectable.
+(function() {
```

Comment:
Popping this into an IIFE for variable scoping

Reply:
@veryspry I dig the thinking! However, since we recommend installing this with our Custom JS plugin, the scope should already be limited: https://github.com/gravitywiz/snippet-library/blob/matt%2Ffix-multi-select-limit/gravity-forms/gw-gravity-forms-custom-js.php#L176-L176

Reply:
I should have noticed since `$ === window.jQuery` in this context :laughing:

Reply:
Fixed!